src/simplewallet/simplewallet.cpp

```diff
@@ -4364,7 +4349,7 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
   // prompt is there is no payment id and confirmation is required
   if (!payment_id_seen && m_wallet->confirm_missing_payment_id())
   {
-     std::string accepted = input_line(tr("No payment id is included with this transaction. Is this okay?  (Y/Yes/N/No): "));
+     std::string accepted = input_line(tr("No payment id is included. An integrated address is recommended instead. Continue anyway?  (Y/Yes/N/No): "));
```

Comment:
Assuming a payment id is going to be used, an integrated address is recommended, but of course, no payment id at all is an option as well. I'll drop this sentence to avoid confusion and squash commits.